packages/mobile/src/web3/reducer.ts

```diff
@@ -71,9 +67,14 @@ export const reducer = (
         ...state,
         commentKey: action.commentKey,
       }
-    case Actions.SET_BLOCK_NUMBER:
+    case Actions.COMPLETE_WEB3_SYNC:
       return {
         ...state,
+        syncProgress: {
+          startingBlock: state.syncProgress.startingBlock,
+          currentBlock: action.latestBlockNumber,
+          highestBlock: action.latestBlockNumber,
+        },
         latestBlockNumber: action.latestBlockNumber,
```

Comment:
Is `latestBlockNumber` still needed?

Reply:
It's used in one place. Not strictly need but I have a feeling we may need it eventually so didn't remove it